lib/pg_ltree/ltree.rb

```diff
@@ -16,6 +16,7 @@ def ltree(column = :path, options: { cascade: true })
 
       if options[:cascade]
         after_update :cascade_update
+        before_destroy :delete_ltree_column_value
```

Comment:
Instead of doing this, you might be able to just call the column directly in the after_destroy callback instead of _was. I think those attributes are frozen on that object.